stable/mongodb/templates/secrets.yaml

```diff
@@ -1,4 +1,4 @@
-{{ if or .Values.usePassword .Values.mongodbPassword -}}
+{{ if and (not .Values.existingSecret) (or .Values.usePassword .Values.mongodbPassword) -}}
```

Comment:
There's no **existingSecret** parameter defined on **values.yml**. Therefore, it won't work.

Reply:
ehm `existingSecret` is also used in `statefulset-*.yaml` also mentioned in `values.yaml` and `values-production.yaml`, so it will(?) - also if a key does not exists in an object/dict it will be simply false.